### PR TITLE
Introduce a script that checks for missing tags for maven packages

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: why_missing_maven/why_missing_maven.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+import daiquiri
+import requests
+import yaml
+
+daiquiri.setup(level=logging.INFO)
+_logger = daiquiri.getLogger(__name__)
+
+
+def iter_files(path):
+    """Yield each file in a directory tree.
```
Comment (rootAvish) [inline why_missing_maven/why_missing_maven.py]: Any particular reason for not using `os.walk`? Would've made this part simpler I think.

Comment (fridex) [reply why_missing_maven/why_missing_maven.py]: You can directly pass a directory tree (it will traverse recursively) or a file.

**Comment by rootAvish** [comment]
Everything else looks good.

**Comment by fridex** [comment]
> Everything else looks good.

@rootAvish feel free to merge in that case

**Pull request merged**